packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -911,6 +915,16 @@ class AnimatedAlign extends ImplicitlyAnimatedWidget {
   /// {@macro flutter.widgets.child}
   final Widget child;
 
+  /// If non-null, sets its height to the child's height multiplied by this factor.
+  ///
+  /// Can be both greater and less than 1.0 but must be positive.
```

Comment:
I may be clearer here and below to say Must be greater than or equal to 0.0, defaults to 1.0.

Reply:
Good point.